low-level/files/opt/seagate/sspl/setup/sspl_test.py

```diff
@@ -25,17 +33,47 @@ class SSPLTestCmd:
     def __init__(self, args: list):
         self.args = args
         self.name = "sspl_test"
+        self.plan = "self_primary"
```

Comment:
Is it default plan? when will it be set to sanity plan in case we required to run sanity.

Reply:
Yes, this is default. No direct run with this script. This should be called from mini provisioner interface like below.
`sspl_setup test --config <gloabl_config_url> --plan sanity`
TestCmd is taking care of args validation.